### ISSUE TITLE
Kubectl preflight --format=yaml results in error that yaml is an unknown output format

### ISSUE THREAD

**Issue by z4ce** [replicatedhq/troubleshoot#905]
**Bug Description**

Running with `kubectl preflight --format=yaml --interactive=false ...`
results in
```
name: cluster-resources    status: running         completed: 0    total: 2
name: cluster-resources    status: completed       completed: 1    total: 2
name: cluster-info         status: running         completed: 1    total: 2
name: cluster-info         status: completed       completed: 2    total: 2
Error: unknown output format: "yaml"
```
**Expected Behavior**
Output should be:
```yaml
pass:
    - title: Required Kubernetes Version
       message: Your cluster meets the recommended and required versions of Kubernetes.
```

**Steps To Reproduce**
Run the preflight plugin with `--format=yaml` worth noting that `--format=json` does work.

**Additional Context**
```
kubectl preflight version
Replicated Preflight 0.49.1
```
MacOS Ventura

**Comment by xavpaice** [comment]
Verified with Troubleshoot v0.51.1 on LInux.

Command used:

```
kubectl preflight --format=yaml --interactive=false --output=/tmp/output https://raw.githubusercontent.com/replicatedhq/troubleshoot/main/config/samples/troubleshoot_v1beta2_preflight.yaml
```

**Comment by CpuID** [comment]
Just a thought: do we want to maintain some consistency with how the output format is specified between tooling?

`kubectl` opts for `-o` or `--output` for `json`/`yaml` etc for basic commands like `get`.

`kubectl support-bundle analyze` also has `--output string   output format: json, yaml` which follows the above approach.

But `kubectl preflight` has 2 CLI args with different usage:

```
--format string                  output format, one of human, json, yaml. only used when interactive is set to false (default "human")
...
-o, --output string                  specify the output file path for the preflight checks
```

...as much as it would be nice to change `--format` to be `--output` for consistency, that would break some backwards compatibility? And what would we then use to supersede the old `--output`? Possibly `-f, --filename`?

**Comment by CpuID** [comment]
YAML output format implemented in https://github.com/replicatedhq/troubleshoot/pull/940 - still under `--format` for now. Do we create a separate GitHub issue relating to my thoughts above about the CLI flag name...?

**Comment by banjoh** [comment]
> YAML output format implemented in #940 - still under `--format` for now. Do we create a separate GitHub issue relating to my thoughts above about the CLI flag name...?

I think we can address this when looking at #933 and #934.
